erdpy/cli_contracts.py

```diff
@@ -284,7 +284,7 @@ def _prepare_sender(args: Any) -> Account:
         sender = LedgerAccount(account_index=args.ledger_account_index, address_index=args.ledger_address_index)
     elif args.pem:
         sender = Account(pem_file=args.pem, pem_index=args.pem_index)
-    elif args.keyfile and args.passfile:
+    elif args.keyfile:
```

Comment:
This change should be applied in several other places, as well (since the logic is currently duplicated in erdpy, unfortunately):
 - `cli_contracts.py` (updated by you)
 - `cli_shared.py`
 - `transactions.py`
 - `staking_provider.py`
 - `validators/core.py`

Reply:
However, how about not passing a `pass_file` at all (whether optional or not) - and pass the `password`, as a string. And possibly add a function in `cli_shared.py`, such as:

```
def load_password(args):
    (loads from file if provided)
    (loads from stdin otherwise)
    returns the password, as a string
```

Then call it as follows:

```
else if args.keyfile is present:
    password = cli_shared.load_password(args)
    account = Account(keyfile = ..., password = password)
```

How do you feel about this change?